### PR TITLE
store/tikv: backoff and retry for 'ServerIsBusy'.

### PR DESCRIPTION
**Pull request by disksing**
@zimulala @coocood @shenli 
Ref: https://github.com/pingcap/tikv/pull/1054

### CODE REVIEW

File: store/tikv/store_test.go [hunk 1]
```diff
@@ -73,6 +76,39 @@ func (s *testStoreSuite) TestOracle(c *C) {
...
+	}()
+
+	go func() {
+		defer wg.Done()
+		txn, err := s.store.Begin()
+		c.Assert(err, IsNil)
+		val, err := txn.Get([]byte("key"))
+		c.Assert(err, IsNil)
+		c.Assert(val, BytesEquals, []byte("value"))
+	}()
```
Comment (coocood) [inline store/tikv/store_test.go]: `SendCopReq` seems not covered.

Comment (disksing) [inline store/tikv/store_test.go]: @coocood Comment addressed.

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**